Upgrade to OpenFOAM 8

Hello,
OpenFOAM 8 was released a few days ago and apparently there a lot of changes, e.g turbulence models directory changed to `MomentumTransportModels ` it will be nice to see this library ported to this version.

Thank you

Yes, I tried to compile it but as expected it will fails:

```
 ./Allwmake
Allwmake /home/SignificantCell/turbinesFoam
wmake libso src
wmakeLnIncludeAll: running wmakeLnInclude on dependent libraries:
    wmakeLnInclude error: base directory /opt/openfoam8/src/transportModels/compressible/ does not exist
    wmakeLnInclude error: base directory /opt/openfoam8/src/TurbulenceModels/turbulenceModels/ does not exist
    wmakeLnInclude error: base directory /opt/openfoam8/src/TurbulenceModels/compressible/ does not exist
g++ -std=c++11 -m64 -Dlinux64 -DWM_ARCH_OPTION=64 -DWM_DP -DWM_LABEL_SIZE=32 -Wall -Wextra -Wold-style-cast -Wnon-virtual-dtor -Wno-unused-parameter -Wno-invalid-offsetof -Wno-attributes -O3  -DNoRepository -ftemplate-depth-100 -I/opt/openfoam8/src/finiteVolume/lnInclude -I/opt/openfoam8/src/meshTools/lnInclude -I/opt/openfoam8/src/sampling/lnInclude -I/opt/openfoam8/src/thermophysicalModels/solidThermo/lnInclude -I/opt/openfoam8/src/transportModels/compressible/lnInclude -I/opt/openfoam8/src/thermophysicalModels/basic/lnInclude -I/opt/openfoam8/src/thermophysicalModels/specie/lnInclude -I/opt/openfoam8/src/TurbulenceModels/turbulenceModels/lnInclude -I/opt/openfoam8/src/TurbulenceModels/compressible/lnInclude -I/opt/openfoam8/src/fvOptions/lnInclude -IlnInclude -I. -I/opt/openfoam8/src/OpenFOAM/lnInclude -I/opt/openfoam8/src/OSspecific/POSIX/lnInclude   -fPIC -fuse-ld=bfd -shared -Xlinker --add-needed -Xlinker --no-as-needed Make/linux64GccDPInt32Opt/interpolations/interpolateUtils.o Make/linux64GccDPInt32Opt/fvOptions/turbineALSource/turbineALSource.o Make/linux64GccDPInt32Opt/fvOptions/crossFlowTurbineALSource/crossFlowTurbineALSource.o Make/linux64GccDPInt32Opt/fvOptions/axialFlowTurbineALSource/axialFlowTurbineALSource.o Make/linux64GccDPInt32Opt/fvOptions/actuatorLineSource/actuatorLineSource.o Make/linux64GccDPInt32Opt/fvOptions/actuatorLineSource/actuatorLineElement/actuatorLineElement.o Make/linux64GccDPInt32Opt/fvOptions/actuatorLineSource/actuatorLineElement/addedMassModel/addedMassModel.o Make/linux64GccDPInt32Opt/fvOptions/actuatorLineSource/actuatorLineElement/dynamicStallModels/dynamicStallModel/dynamicStallModel.o Make/linux64GccDPInt32Opt/fvOptions/actuatorLineSource/actuatorLineElement/dynamicStallModels/LeishmanBeddoes/LeishmanBeddoes.o Make/linux64GccDPInt32Opt/fvOptions/actuatorLineSource/actuatorLineElement/dynamicStallModels/LeishmanBeddoes3G/LeishmanBeddoes3G.o Make/linux64GccDPInt32Opt/fvOptions/actuatorLineSource/actuatorLineElement/dynamicStallModels/LeishmanBeddoesSGC/LeishmanBeddoesSGC.o Make/linux64GccDPInt32Opt/fvOptions/actuatorLineSource/actuatorLineElement/dynamicStallModels/LeishmanBeddoesSD/LeishmanBeddoesSD.o Make/linux64GccDPInt32Opt/fvOptions/actuatorLineSource/actuatorLineElement/profileData/profileData.o -L/opt/openfoam8/platforms/linux64GccDPInt32Opt/lib \
    -lfiniteVolume -lsampling -lmeshTools -lturbulenceModels -lcompressibleTurbulenceModels -lfvOptions  -o /home/SignificantCell/OpenFOAM/SignificantCell-8/platforms/linux64GccDPInt32Opt/lib/libturbinesFoam.so
/usr/bin/ld.bfd: cannot find -lturbulenceModels
/usr/bin/ld.bfd: cannot find -lcompressibleTurbulenceModels
collect2: error: ld returned 1 exit status
make: *** [/opt/openfoam8/wmake/makefiles/general:169: /home/SignificantCell/OpenFOAM/SignificantCell-8/platforms/linux64GccDPInt32Opt/lib/libturbinesFoam.so] Error 1

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,7 +5,6 @@ dist: xenial
 language: cpp
 
 env:
-    - OF_VERS=5
     - OF_VERS=6
     - OF_VERS=7
     - OF_VERS=8

--- a/.travis.yml
+++ b/.travis.yml
@@ -8,6 +8,7 @@ env:
     - OF_VERS=5
     - OF_VERS=6
     - OF_VERS=7
+    - OF_VERS=8
 
 before_install:
     - sudo add-apt-repository http://dl.openfoam.org/ubuntu

--- a/README.md
+++ b/README.md
@@ -5,7 +5,6 @@ turbinesFoam
 ![OpenFOAM 8](https://img.shields.io/badge/OpenFOAM-8-brightgreen.svg)
 ![OpenFOAM 7](https://img.shields.io/badge/OpenFOAM-7-brightgreen.svg)
 ![OpenFOAM 6](https://img.shields.io/badge/OpenFOAM-6-brightgreen.svg)
-![OpenFOAM 5.x](https://img.shields.io/badge/OpenFOAM-5.x-brightgreen.svg)
 [![DOI](https://zenodo.org/badge/4234/turbinesFoam/turbinesFoam.svg)](https://zenodo.org/badge/latestdoi/4234/turbinesFoam/turbinesFoam)
 
 turbinesFoam is a library for simulating wind and marine hydrokinetic turbines

--- a/README.md
+++ b/README.md
@@ -2,6 +2,7 @@ turbinesFoam
 ============
 
 [![Build Status](https://travis-ci.org/turbinesFoam/turbinesFoam.svg?branch=master)](https://travis-ci.org/turbinesFoam/turbinesFoam)
+![OpenFOAM 8](https://img.shields.io/badge/OpenFOAM-8-brightgreen.svg)
 ![OpenFOAM 7](https://img.shields.io/badge/OpenFOAM-7-brightgreen.svg)
 ![OpenFOAM 6](https://img.shields.io/badge/OpenFOAM-6-brightgreen.svg)
 ![OpenFOAM 5.x](https://img.shields.io/badge/OpenFOAM-5.x-brightgreen.svg)

--- a/src/Make/options
+++ b/src/Make/options
@@ -2,18 +2,10 @@ EXE_INC = \
     -I$(LIB_SRC)/finiteVolume/lnInclude \
     -I$(LIB_SRC)/meshTools/lnInclude \
     -I$(LIB_SRC)/sampling/lnInclude \
-    -I$(LIB_SRC)/thermophysicalModels/solidThermo/lnInclude \
-    -I$(LIB_SRC)/transportModels/compressible/lnInclude \
-    -I$(LIB_SRC)/thermophysicalModels/basic/lnInclude \
-    -I$(LIB_SRC)/thermophysicalModels/specie/lnInclude \
-    -I$(LIB_SRC)/TurbulenceModels/turbulenceModels/lnInclude \
-    -I$(LIB_SRC)/TurbulenceModels/compressible/lnInclude \
     -I$(LIB_SRC)/fvOptions/lnInclude
 
 LIB_LIBS = \
     -lfiniteVolume \
     -lsampling \
     -lmeshTools \
-    -lturbulenceModels \
-    -lcompressibleTurbulenceModels \
     -lfvOptions

